tests/Unit/Utils/PhoneTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Tests\Unit\Utils;
+
+use App\Utils\Phone;
+use PHPUnit\Framework\TestCase;
+
+class PhoneTest extends TestCase
```

Comment:
On a pris l'habitude de factoriser les tests en utilisant les générateurs php + la feature dataprovider de phpunit, tu as pleins dans toutes les classes utilitaires
https://docs.phpunit.de/en/9.6/writing-tests-for-phpunit.html#data-providers